src/data/CHDatasource.ts

```diff
@@ -202,6 +202,61 @@ export class Datasource
     };
   }
 
+  // REMOVE this whole method
+  // getDataProvider(type: SupplementaryQueryType, request: DataQueryRequest<CHQuery>): Observable<DataQueryResponse> | undefined { ... }
+
+  // ADD this instead:
```

Comment:
but we didnt remove a function called `getDataProvider` right?

Do we need to keep this comment even if we did?